fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/FedoraSession.java

```diff
@@ -77,16 +80,31 @@
     Map<String, String> getNamespaces();
 
     /**
-     * Set session-specific data
+     * Add session-specific data
```

Comment:
Can we have a note here that it is up to the impl whether multiple values per key are afforded, and a note on the immediate impl that they aren't?